### DISCUSSION TITLE
Other Launchers not Launching when i click play

### DISCUSSION THREAD

**Discussion by Doogel** [moraroy/NonSteamLaunchers-On-Steam-Deck#246] [Q&A]
Hi,
All the Launchers (EA, Epic, Xbox) are installing correctly and are showing up in Steam but are not launching, neither in desktop mode nore in Gaming Mode.
I tried to change the Compability to experimental or other versions. 
When i click play in Steam it looks like it is about to start but nothing happens.

Could anyone be so kind and help with this?

**Comment by moraroy** [comment]
@Doogel could be a number of reasons. Without knowing more details I can't really do much. Xbox doesn't need proton so please do not force that on the websites. Google Chrome needs to be installed in order to launch the websites. As far as epic is concerned, that should work no problem. The ea app opens right now but a recent update may have broken it.

**Reply by Doogel** [reply]
I just tried to reinstall and set up the apps as described in the instruction and still no luck, i guess i need a bit more tinkering or i will try an older version

**Reply by moraroy** [reply]
@Doogel make sure your running the .desktop file on the desktop.

**Reply by troney07** [reply]
I'm having the same issue, they are all there but they won't launch.  I will admit that the first time I tried I ran it from my downloads but I have since moved to my desktop and made sure that the launcher ended in ".desktop".  

I saw in a video that someone had to set a "Steam Deck User" password before it would allow them to install "deckyLoader" correctly. taking a shot in the dark here.. is there any assumption that we have already done that before running "NonSteam Launchers" ?

I understand that all this is vague and not much you can do to attempt at fixing my problem but i'll keep trying to get it to work...

**Reply by moraroy** [reply]
@troney07 what do you mean by they are all there? Do you mean in your Steam library? If so, you're saying they won't launch? Is proton compatability checked? If it is, your proton may be corrupted can you reinstall the latest proton by downloading and isntalling Proton Up Qt. And reinstalling the latest proton.

**Reply by troney07** [reply]
Yes they are in my steam library and yes they won't launch, no error message no nothing. Yes Proton Compatability is checked. I'll do as you suggest now and let you know. Thank you,
